privacy/ccpa/policy_test.go

```diff
@@ -303,38 +546,74 @@ func TestValidateConsent(t *testing.T) {
 func TestShouldEnforce(t *testing.T) {
 	testCases := []struct {
 		description string
+		bidder      string
 		policy      Policy
 		expected    bool
 	}{
 		{
 			description: "Enforceable",
+			bidder:      "a",
 			policy:      Policy{Value: "1-Y-"},
 			expected:    true,
 		},
+		{
+			description: "Enforceable - No Sale For Different Bidder",
+			bidder:      "a",
+			policy:      Policy{Value: "1-Y-", NoSaleBidders: []string{"b"}},
+			expected:    true,
+		},
 		{
 			description: "Not Enforceable - Not Present",
+			bidder:      "a",
 			policy:      Policy{Value: ""},
 			expected:    false,
 		},
 		{
 			description: "Not Enforceable - Opt-Out Unknown",
+			bidder:      "a",
 			policy:      Policy{Value: "1---"},
 			expected:    false,
 		},
 		{
 			description: "Not Enforceable - Opt-Out Explicitly No",
+			bidder:      "a",
 			policy:      Policy{Value: "1-N-"},
 			expected:    false,
 		},
+		{
+			description: "Not Enforceable - No Sale All Bidders",
+			bidder:      "a",
+			policy:      Policy{Value: "1-Y-", NoSaleBidders: []string{"*"}},
+			expected:    false,
+		},
+		{
+			description: "Not Enforceable - No Sale All Bidders Mixed With Specific Bidders",
+			bidder:      "a",
+			policy:      Policy{Value: "1-Y-", NoSaleBidders: []string{"b", "*", "c"}},
+			expected:    false,
+		},
+		{
+			description: "Not Enforceable - No Sale Specific Bidder",
+			bidder:      "a",
+			policy:      Policy{Value: "1-Y-", NoSaleBidders: []string{"a"}},
+			expected:    false,
+		},
+		{
+			description: "Not Enforceable - No Sale Specific Bidder Case Insensitive",
```

Comment:
Should it be case insensitive?

Reply:
If it isn't too much work. I don't believe there should exist two bidders that differ only by case. So it is a nice feature if we "correct" for mis-cased bidders. On the other hand we do have other places where we are sensitive to the case of bidder names.

Reply:
It's not much trouble at all. Simply a `strings.EqualsFold`. What other places are we  sensitive to the case of bidder name?